packages/eui/cypress/support/setup/mount.tsx

```diff
@@ -29,7 +29,7 @@ const mountCommand = (
   options: MountOptions = {}
 ): ReturnType<typeof mount> => {
   const { providerProps } = options;
-  return cypressMount(<EuiProvider {...providerProps}>{children}</EuiProvider>);
+  return cypressMount(<EuiProvider {...providerProps} colorMode="LIGHT">{children}</EuiProvider>);
```

Comment:
We should allow individual tests to override/enforce a dark mode if needed.

```suggestion
  return cypressMount(<EuiProvider colorMode="LIGHT" {...providerProps}>{children}</EuiProvider>);
```

Also, just wondering, do we want to force the light color mode for every test or just for the ones that assert on styles? 🤔

Reply:
>We should allow individual tests to override/enforce a dark mode if needed.
That's fair, I was maybe being a bit too strict here wanting to force it having run into the issue myself 😬 

>Do we want to force the light color mode for every test or just for the ones that assert on styles?
For now, while we don't have any differentiation between LIGHT and DARK mode in test assertions or testing system defaults, I'd say let's fix it. Would there be a benefit in manually adding it only for style tests? Imho, it would seem more overhead to remember adding it? 🤔

Reply:
Yeah I dunno if there's a super strong benefit! It'd probably be a very minor developer experience thing for EUI devs who prefer dark mode and are writing or debugging Cypress tests, and don't want to be blasted in the face with light mode screens locally?

Reply:
Hmm, yeah I get your point. Though as a dark mode user myself I haven't yet been bothered by the tests in light mode yet. Especially since the cypress UI is light anyway and only the embedded EUI content would be dark 😅 

It might be fair to keep it as is for now and address it further if we notice the need? 🤔

Reply:
Ahhh great point! Yep, let's keep it as-is then!